x-pack/plugins/search_inference_endpoints/public/components/all_inference_endpoints/render_table_columns/table_columns.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { InferenceAPIConfigResponse } from '@kbn/ml-trained-models-utils';
+import React from 'react';
+import * as i18n from '../../../../common/translations';
+import { useActions } from './render_actions/use_actions';
+import { RenderEndpoint } from './render_endpoint/render_endpoint';
+import { ServiceProvider } from './render_service_provider/service_provider';
+import { TaskType } from './render_task_type/task_type';
+import { DeploymentStatus } from './render_deployment_status/deployment_status';
+import { DeploymentStatusEnum, ServiceProviderKeys, TaskTypes } from '../types';
+
+export const useTableColumns = () => {
+  const { actions } = useActions();
+
+  const TABLE_COLUMNS = [
+    {
+      field: 'deployment',
+      name: '',
+      render: (deployment: DeploymentStatusEnum) => {
+        if (deployment != null) {
+          return <DeploymentStatus status={deployment} />;
+        }
+
+        return null;
+      },
+      width: '16px',
```

Comment:
Let's change this to "32px" and add a new line for 
`align: 'center'`

Reply:
Updated